sci-libs/vtk/files/vtk-8.2.0-pugixml.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/IO/CityGML/CMakeLists.txt b/IO/CityGML/CMakeLists.txt
```

Comment:
Don't forget http://dev.gentoo.org/~vapier/clean-patches! (Include a bug link and remove git rubbish).

Reply:
done!